src/main/java/com/tw/go/plugin/provider/github/GitHubProvider.java

```diff
@@ -82,4 +77,42 @@ public User getUser(Profile profile) {
         }
         return users;
     }
+
+    @Override
+    public boolean authorize(PluginSettings pluginSettings, User user) {
+        if(StringUtils.isEmpty(pluginSettings.getUsernameRegex())) {
+            return true;
+        } else {
+            return isAMemberOfOrganization(pluginSettings, user);
+        }
+    }
+
+    private boolean isAMemberOfOrganization(PluginSettings pluginSettings, User user) {
+        boolean result = false;
+        try {
+            GitHub github = getGitHub(pluginSettings);
+
+            GHOrganization organization = github.getOrganization(pluginSettings.getUsernameRegex());
```

Comment:
We can split by comma or use another common trick to do this, but the GH token / user password must have access to private membership of organization to call `ghUser.isMemberOf(organization)`. Another option is fallback to public information, but we should not rely on this information.
One last thing we have to keep in mind is that every verification is a http request to GH api.

I think support 1 organization is enough for most of usages and we can improve in the future if somebody ask.

Btw, we should rename the field "username_regex" if you are interest in merge this PR.